tools/mti-utils/process_intensities.xml

```diff
@@ -74,6 +91,14 @@ quant.to_csv(os.path.join(cwd, 'processed_quant.csv'))
             </when>
             <when value="SBR">
                 <param name="AF_col" type="text" value="AF_channel" label="Name of column in markers file containing respective AF channel for each marker" />
+                <conditional name="filter">
+                    <param name="AF_filter" type="select" label="Select whether to clip or filter out AF values equal to 0">
```

Comment:
Why not just a select? The conditional seems not needed

Reply:
Thanks for catching that, @bgruening!